### ISSUE TITLE
[jhelp] Glossary Joomla 4

### ISSUE THREAD

**Issue by YGomiero** [joomla/joomla-websites#1662]
Joomla 4 :
Do not translate the line in the joomla.ini file 
administrator/language/xx-XX/joomla.ini

JHELP_GLOSSARY="Glossary"

With the current version (J4.0.2), there is an error in the help section of Joomla 4 because the translation term is included in the url of jdoc, necessarily on a page that does not exist.

The problem also occurs on all translation packs.

line 35 : administrator\components\com_admin\tmpl\help\default.php

`<?php echo HTMLHelper::_('link', Help::createUrl('JHELP_GLOSSARY'), Text::_('COM_ADMIN_GLOSSARY'), ['target' => 'helpFrame']); ?>`

**Comment by conconnl** [comment]
Correct, that is why they are hidden and also mark as "DONOTTRANSLATE"
For some reason they are automatically translated.
Need to find the correct way to stop this, without increasing the work for translators.
cc @HLeithner

**Comment by conconnl** [comment]
@YGomiero I checked Joomla 3 translations as well.
This string is also translated in that version for various languages for years.
Doesn't that break J3?
If not, why....?
